GUI/Main.Designer.cs

```diff
@@ -1047,9 +1047,8 @@ private void InitializeComponent()
             this.RecommendedDialogLabel.Name = "RecommendedDialogLabel";
             this.RecommendedDialogLabel.Size = new System.Drawing.Size(627, 20);
             this.RecommendedDialogLabel.TabIndex = 6;
-            this.RecommendedDialogLabel.Text = "The following modules have been recommended by one or more of the chosen modules:" +
-    "";
             // 
+            this.RecommendedDialogLabel.Text = "The following modules have been recommended or suggested by one or more of the chosen modules:";
```

Comment:
The tab is missing a legend I think. It's probably hard to guess which mods are recommended and which are suggested if you don't know the checked-unchecked rule.
Changing this sentence to `The following modules have been recommended (mostly checked - upper part) or suggested (unchecked - lower part).` maybe would suffice.

Reply:
Apart from that, the changes look good to me. And they work!

Reply:
How about some `ListViewGroup`s instead?

![image](https://user-images.githubusercontent.com/1559108/56872207-dad8a780-6a15-11e9-88a4-941971bda1ce.png)

Reply:
Even better! 🎉